modules/oc-mirror-oci-format.adoc

```diff
@@ -5,12 +5,25 @@
 
 :_content-type: PROCEDURE
 [id="oc-mirror-oci-format_{context}"]
-= Mirroring file-based catalog Operator images in OCI format
+= Including local OCI Operator catalogs
 
-You can use the oc-mirror plugin to mirror Operators in the Open Container Initiative (OCI) image format, instead of Docker v2 format. You can copy Operator images to a file-based catalog on disk in OCI format. Then you can copy local OCI images to your target mirror registry.
+While mirroring {product-title} releases, Operator catalogs, and additional images from a registry to a partially disconnected cluster, you can include Operator catalog images from a local file-based catalog on disk. The local catalog must be in the Open Container Initiative (OCI) format.
 
-:FeatureName: Using the oc-mirror plugin to mirror Operator images in OCI format
-include::snippets/technology-preview.adoc[]
+The local catalog and its contents are mirrored to your target mirror registry based on the filtering information in the image set configuration file.
+
+[IMPORTANT]
+====
+When mirroring local OCI catalogs, any {product-title} releases or additional images that you want to mirror with the local OCI-formatted catalog must be pulled from a registry.
```

Comment:
Maybe use `along with the local OCI-formatted` ?
Just in case they understand this as being " with flag/feature local OCI-formatted"

Really not sure. up to you